packages/beacon-node/src/util/dataColumns.ts

```diff
@@ -192,3 +194,95 @@ export function getDataColumns(nodeId: NodeId, custodyGroupCount: number): Colum
     .flatMap(computeColumnsForCustodyGroup)
     .sort((a, b) => a - b);
 }
+
+/**
+ * Computes the cells for each blob and combines them with cell proofs.
+ * Similar to the computeMatrix function described below.
+ *
+ * SPEC FUNCTION (note: spec currently computes proofs, but we already have them)
+ * https://github.com/ethereum/consensus-specs/blob/dev/specs/fulu/das-core.md#compute_matrix
+ */
+export function getCellsAndProofs(blobs: fulu.BlobAndProofV2[]): [Uint8Array[], Uint8Array[]][] {
+  return blobs.map((blob) => {
+    const cells = ckzg.computeCells(blob.blob);
+    const proofs = blob.proofs;
+    return [cells, proofs];
+  });
+}
```

Comment:
For future maintainers I think we want to be more specific here about which proofs are getting paired.  I had to do a double take so other may also in the future.

```suggestion
export function getCellsAndProofs(blobBundles: fulu.BlobAndProofV2[]): [Uint8Array[], Uint8Array[]][] {
  return blobs.map(([blob, cellProofs]) => {
    const cells = ckzg.computeCells(blob.blob);
    return [cells, cellProofs];
  });
}
```

Reply:
Pushed an update for this, but since BlobAndProof is an object, I had to destructure it and transform the name. Think we should update `BlobAndProofV2` to use `cellProofs` instead of `proofs`, since that's a Lodestar type?

Reply:
oh snap... good point. was sudo-coding in the PR and am incorrect (I thank god for intellisense everyday lol).   Here is an update that "should" be closer.  Feel free to massage as appropriate if its not 100% accurate

```ts
export function getCellsAndProofs(blobBundles: fulu.BlobAndProofV2[]): [Uint8Array[], Uint8Array[]][] {
  return blobs.map(({blob, proofs}) => {
    const cells = ckzg.computeCells(blob);
    return [cells, proofs];
  });
}
```

Reply:
sounds good, take a look at what I've got and let me know if you prefer yours!